tests/test_minimum_clearance.py

```diff
@@ -0,0 +1,33 @@
+"""
+Tests for the minimum clearance property.
+"""
+
+import math
+
+import pytest
+
+from shapely.wkt import loads as load_wkt
+from shapely.geos import geos_version
+
+
+pytestmark = pytest.mark.skipif(geos_version < (3, 6, 0), reason='GEOS 3.6.0 required')
```

Comment:
It looks like the tests are being skipped even on versions of GEOS >= 3.6.0. Did I set this up correctly?